lib/event.rb

```diff
@@ -0,0 +1,71 @@
+require 'pry'
+
+class Event
+
+  attr_reader :name, :ages
+
+  def initialize(name, ages)
+    @name = name
+    @ages = ages
+  end
+
+  def print_standard_deviation(ages)
+    ages = self.ages
+    avg = average(ages)
+    lesser_array = values_sub_avg(avg, ages)
+    squared_array = values_squared(lesser_array)
+    pre_final_value = squared_array.sum / ages.count
+    standard_deviation = Math.sqrt(pre_final_value).round(2)
+    p standard_deviation
+  end
+
+  def average(numeric_values)
+    sum = numeric_values.sum
+    count = numeric_values.count
+    avg = sum.to_f / count.to_f
+  end
+
+  def values_sub_avg(avg, numeric_values)
+    lesser_array = []
+    numeric_values.each do |num|
+    lesser_num = num - avg
+    lesser_num = lesser_num.round(2)
+    lesser_array << lesser_num
+  end
```

Comment:
the code inside a block should be indented, so this method should look like this:

```
  def values_sub_avg(avg, numeric_values)
    lesser_array = []
    numeric_values.each do |num|
        lesser_num = num - avg
        lesser_num = lesser_num.round(2)
        lesser_array << lesser_num
    end
    lesser_array
  end
```